packages/dashboard/e2e/tests/utils/resourceExplorer.ts

```diff
@@ -77,6 +77,29 @@ export const resourceExplorerUtil = (page: Page) => {
     },
   };
 
+  const modeledActions = {
+    findAsset: (label: string) => {
+      return frame.getByLabel(`Select asset ${label}`);
+    },
+    selectAsset: (label: string) => {
+      return frame.getByLabel(`Select asset ${label}`).click();
+    },
+    findProperty: (label: string) => {
+      return frame.getByLabel(`Select modeled data stream ${label}`);
+    },
+    selectProperty: async (label: string) => {
+      await frame.getByLabel(`Select modeled data stream ${label}`).click();
+    },
+    findDisabledProperty: (label: string) => {
+      return frame.getByLabel(
+        `${label} data not supported for the selected widget`
+      );
+    },
+    addToWidget: async () => {
```

Comment:
a lot of these are the same as the ones in `assetModelActions` above 
could we pull out any shared utils so we dont duplicate the actions